src/components/game.js

```diff
@@ -85,12 +250,36 @@ class Game extends React.Component {
     return board;
   }
 
+  getTile = (width, row) => {
+    const { board } = this.state;
+    const rows = [];
+    for (let i = 0; i < width; i += 1) {
+      rows[i] = <Tile key={i} value={board[row][i]} />;
+    }
+
+    return (
+      <Row key={uuid.v4()}>
+        {rows}
+      </Row>
+    );
+  }
+
+  getState = (width, height) => {
```

Comment:
getState sugeruje, że to jest funkcja powiązana z setState, potencjalnie mylące.
może `getBoard`, albo `getBoardTiles` albo jeszcze coś innego?

Reply:
zmienione